packages/node_modules/@webex/react-component-confirmation-modal/src/__snapshots__/index.test.js.snap

```diff
@@ -3,7 +3,10 @@
 exports[`ConfirmationModal component renders properly 1`] = `
 <div>
   <div
+    aria-modal="true"
```

Comment:
Had to make changes to the snapshot in order for tests to pass.

Reply:
Did you manually fixed it or used the command?

Reply:
Had to manually add it... tried the -u command but it didn't update the code.